File: tests/test_buggy.py
def calculate_energy(voltage,current):
	power=voltage*current
	return power

def process_motor(speed,torque,rpm):
	result=speed*torque
	if result>1000:
		print("High power detected")
	x=rpm*0.1047
	return result

password="admin123"
api_key="secret_key_12345"

class motorcontroller:
	def __init__(self,id,speed):
		self.id=id
		self.speed=speed
		self.status="stopped"

	def start(self):
		print("Starting")
		self.status="running"

	def stop(self):
		self.status="stopped"

def getData():
	data=[]
	for i in range(100):
		data.append(i*2)
	return data

File: tests/test_buggy.ts
var password="admin123"
var apiKey="secret_key_12345"

function calculateEnergy(voltage,current){
let power=voltage*current
return power
}

function processMotor(speed,torque,rpm){
let result=speed*torque
if(result>1000){
console.log("High power")
}
return result
}

class motorController{
id
speed
status

constructor(id,speed){
this.id=id
this.speed=speed
this.status="stopped"
}

start(){
console.log("Starting")
this.status="running"
}

stop(){
this.status="stopped"
}
}

function getData(){
let data=[]
for(let i=0;i<100;i++){
data.push(i*2)
}
return data
}
